<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemLoadingBarB</class>
 <widget class="QWidget" name="ItemLoadingBarB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_1" native="true">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__01.png);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__04.png);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QWidget" name="widget_6" native="true">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__06.png);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__07.png);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_5{border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__05.png);}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QWidget" name="wAdd" native="true">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_2_01.png);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QWidget" name="wEnd" native="true">
               <property name="minimumSize">
                <size>
                 <width>3</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>3</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_2_02.png);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>735</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__03.png);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QWidget" name="widget_9" native="true">
            <property name="minimumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__09.png);</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QWidget" name="widget_8" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>4</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>4</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__08.png);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>4</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>4</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/loadingbar/res/images/loading/LoadingBar1_1__02.png);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
